### PR TITLE
[Feature] Support gather model metrics

### PR DESCRIPTION
**Pull request by hhaAndroid**
Support gather model metrics

### CODE REVIEW

File: .dev_scripts/convert_benchmark_script.py [hunk 1]
```diff
@@ -11,6 +11,8 @@ def parse_args():
     parser.add_argument(
         'json_path', type=str, help='json path output by benchmark_filter')
     parser.add_argument('partition', type=str, help='slurm partition name')
+    parser.add_argument(
```
[the remark this answers is not in the document]
Comment (hhaAndroid) [reply .dev_scripts/convert_benchmark_script.py:14]: I think it's ok

**Comment by jshilong** [comment]
This pr is fine to me.

File: .dev_scripts/gather_benchmark_metric.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            # 1 read config
+            cfg = mmcv.Config.fromfile(config)
+            total_epochs = cfg.total_epochs
+
+            final_results = cfg.evaluation.metric
+            if not isinstance(final_results, list):
+                final_results = [final_results]
+
+            final_results_out = []
+            for key in final_results:
```
Comment (jshilong) [inline .dev_scripts/gather_benchmark_metric.py:48]: Check if there are other metrics

**Review by jshilong — APPROVED** [review]

File: .dev_scripts/gather_benchmark_metric.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+if __name__ == '__main__':
+    args = parse_args()
+
+    root_path = args.root
+    benchmark_json_path = args.benchmark_json
+
+    model_configs = mmcv.load(benchmark_json_path)['models']
+    result_dict = {}
+    for config in model_configs:
+
```
Comment (ZwwWayne) [inline .dev_scripts/gather_benchmark_metric.py:32]: clean unnecessary empty line.

File: .dev_scripts/gather_benchmark_metric.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        config_name = osp.split(config)[-1]
+        config_name = osp.splitext(config_name)[0]
+        result_path = osp.join(root_path, config_name)
+
+        if osp.exists(result_path):
+
```
Comment (ZwwWayne) [inline .dev_scripts/gather_benchmark_metric.py:38]: Clean unnecessary empty line

File: .dev_scripts/gather_benchmark_metric.py [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+                    continue
+                result_dict[config] = model_performance
+            else:
+                print(f'{config} not exist: {ckpt_path}')
+
+        else:
+            print(f'not exist: {config}')
+
+    # 4 print results
+    print('===================================')
```
Comment (ZwwWayne) [inline .dev_scripts/gather_benchmark_metric.py:75]: May dump it into a json file rather than simply print them

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**